config/env.development

```diff
@@ -28,7 +28,7 @@ API_VERSION=v1
 # Auth Service
 ################################################################################
 
-# Auth Service Port (default is 4444)
+# Auth Service Port (default is 7777)
```

Comment:
Doesn't feed discovery use 7777?

Reply:
It did, but I've now updated it.  We had two servers on the same port (auth was always on 7777 so it wasn't available when Tony chose it).